### ISSUE TITLE
Conflict between LifterLMS Quizzes and Starter Templates plugin

### ISSUE THREAD

**Issue by nrherron92** [gocodebox/lifterlms#1030]
### Reproduction Steps

Install LifterLMS and Starter Templates (https://wordpress.org/plugins/astra-sites/)
Go into the Course Builder and add a quiz.

### Expected Behavior

The quiz in the Course Build should load to allow you to edit the settings.

### Actual Behavior

The spinning wheel of death keeps loading and loading so you can't edit the settings.

### Error Messages / Logs

JS error:

```
Uncaught TypeError: wp.editor.remove is not a function
    at i.render_editor (SettingsFields.js:199)
    at i.<anonymous> (SettingsFields.js:180)
    at underscore.min.js?ver=1.8.3:1
    at Function.v.each.v.forEach (underscore.min.js?ver=1.8.3:1)
    at i.render (SettingsFields.js:179)
    at i.render_subview (_Subview.js:174)
    at i.render (Quiz.js:143)
    at i.add_new_quiz (Quiz.js:251)
    at HTMLElement.dispatch (load-scripts.php?c=1&load[chunk_0]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=5.3.2:3)
    at HTMLElement.r.handle (load-scripts.php?c=1&load[chunk_0]=jquery-core,jquery-migrate,utils,moxiejs,plupload&ver=5.3.2:3)

```

HS tickets: HS-112672, HS-112814, HS-112845, HS-113000

**Comment by thomasplevy** [comment]
Astra Sites loads the `@wordpress/wp-editor` module as a dependency which overwrites the "old" `wp.editor` instance that would provide access to TinyMCE

It looks like WordPress added a reference to the TinyMCE editor via `wp.oldEditor` during the 5.0 release: 

https://github.com/WordPress/WordPress/blob/556c6d80d81afadc377f7382e8344b79b1f71da6/wp-includes/script-loader.php#L365-L371

It should be safe to switch our builder to reference only the oldEditor but I'm going to work on a fallback to use `wp.editor` if `wp.oldEditor` doesn't exist (this would allow us to continue supporting 4.9 and lower, even though we don't really need to. It's a nice safety measure to take.

**Comment by thomasplevy** [comment]
@nrherron92 fixed in 3.37.11, please update related HS tickets. Thanks
